doc/manual/redirects.js

```diff
@@ -19,6 +19,7 @@ const redirects = {
     "chap-distributed-builds": "advanced-topics/distributed-builds.html",
     "chap-post-build-hook": "advanced-topics/post-build-hook.html",
     "chap-post-build-hook-caveats": "advanced-topics/post-build-hook.html#implementation-caveats",
+    "chap-writing-nix-expressions": "language/index.html",
```

Comment:
```suggestion
    "chap-writing-nix-expressions": "language",
```

perhaps

Reply:
This whole piece is rather cargo-cult-y, but leaving the file names explicit keeps it independent of server-side URL rewriting capabilities. Maybe it doesn't matter even in the long run, it sure doesn't for now since the official stuff is served through Netlify that *does* rewrite URLs, but let's leave it for consistency.

Reply:
I was wrong. The menu links to

```suggestion
    "chap-writing-nix-expressions": "language/",
```

and it seems safe to assume that that is the canonical URL for it.

Anything that isn't the canonical URL is inconsistent and should be fixed - not used as an excuse.

Reply:
Let's do that in a separate commit then.